src/reference/command_output_tags.adoc

```diff
@@ -0,0 +1,37 @@
+== Command Output Tags
+
+Commands can print tags to stdout to communicate with Cog. To use one, print it
+to stdout at the beginning of a newline before returning your actual response.
```

Comment:
`JSON` is a weird one which breaks that rule. But I think an example (as you mentioned) would clear this up pretty well.